src/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -53,8 +38,10 @@ public SafeDeleteNegoContext(SafeFreeNegoCredentials credential, string targetNa
             Debug.Assert((null != credential), "Null credential in SafeDeleteNegoContext");
             try
             {
-                _targetNameKerberos = SafeGssNameHandle.CreateTarget(targetName, isNtlmTarget: false);
-                _targetNameNtlm = SafeGssNameHandle.CreateTarget(targetName, isNtlmTarget: true);
+                // Convert any "SERVICE/HOST" style of targetName to use "SERVICE@HOST" style.
+                // This is because the System.Net.Security.Native GSS-API layer uses
+                // GSS_C_NT_HOSTBASED_SERVICE format for targetName.
+                _targetName = SafeGssNameHandle.CreateTarget(targetName.Replace('/', '@'));
```

Comment:
Is there any situation where `/` might appear where it shouldn't be replaced? e.g. could `/` be escaped somehow?

Reply:
No. The formats for SPN are very specific.  Windows always uses SERVICE/HOST for example for Service Principal Names.  And there can't be embedded slashes in those.  It would be a malformed name and generate errors anyways.